win-odbc64/odbcclient/drvr35/cstmt.cpp

```diff
@@ -1063,7 +1063,20 @@ SQLRETURN CStmt::SendSQLCommand(BOOL SkipProcess, SQLCHAR *StatementText,
             else
                 m_StmtType = TYPE_UNKNOWN;
         }
-
+		else if (strcmp(token, "SET") == 0)

```

Comment:
use space pls!
else the format looks not good.

Reply:
Thanks for your suggestion, I have corrected this problem.